plugins/discourse-ai/app/jobs/regular/detect_translate_topic.rb

```diff
@@ -50,7 +50,11 @@ def execute(args)
 
       locales.each do |locale|
         next if LocaleNormalizer.is_same?(locale, detected_locale)
-        next if topic.localizations.matching_locale(locale).exists?
+        exists = topic.localizations.matching_locale(locale).exists?
+
+        if exists && !DiscourseAi::Translation::TopicLocalizer.has_relocalize_quota?(topic, locale)
+          next
+        end
```

Comment:
Styling: how about extracing the `DiscourseAi::Translation::TopicLocalizer.has_relocalize_quota?(topic, locale)`  into a variable and then do `if exists && !has_quota` or something similar?